app/javascript/dashboard/components/widgets/conversation/Message.vue

```diff
@@ -197,11 +197,15 @@ export default {
         'is-private': this.data.private,
         'is-image': this.hasImageAttachment,
         'is-text': this.hasText,
+        'is-from-bot': this.isSentByBot,
       };
     },
     isPending() {
       return this.data.status === MESSAGE_STATUS.PROGRESS;
     },
+    isSentByBot() {
+      return !this.sender.type || this.sender.type === 'bot';
```

Comment:
sender can be null

Reply:
@nithindavid Sorry about this, this was not needed. This was handled already in the computed property.